src/commands/style/detector.css

```diff
@@ -0,0 +1,386 @@
+html {
```

Comment:
💡 self note: this whole `css` file need triming to much smaller minimilistic version.